articles/quickstart/native/ionic4/01-login.md

```diff
@@ -193,6 +196,7 @@ export class AppComponent {
 
       // Redirect back to app after authenticating
       (window as any).handleOpenURL = (url: string) => {
+        this.auth.safariViewController.hide();
```

Comment:
I think the auth service should not expose the safari view controller directly. Instead, a new method `hideBrowser` or similar could be created and the property remain _private_. 

Also, after checking the sample code, wouldn't it be better to call `hide()` after you get an event from that browser you just opened? if that's something that you receive and can relate to this scenario..

See 

https://github.com/auth0-samples/auth0-ionic4-samples/pull/6/files#diff-fab356ab6389b09c66727fb44caa49ccL80-L87

Reply:
I agree @lbalmaceda , I had toyed with that solution, but opted to default to how the Ionic 3 sample we had was handling it.  After getting a second pair of eyes on it, definitely think we should go the route you are suggesting